react/MuiCozyTheme/Dialog/DialogContent.jsx

```diff
@@ -0,0 +1,3 @@
+import DialogContent from '@material-ui/core/DialogContent'
+
```

Comment:
because we plan to override them pretty soon?

Reply:
No, just because we use them and we'd rather import everything from cozy-ui rather than mix cozy-ui and MUI imports.